packages/dendrites/src/nupic/research/frameworks/dendrites/mixins/prototype_context.py

```diff
@@ -21,7 +21,9 @@
 
 import abc
 
+import numpy as np
 import torch
+from scipy.stats import f
```

Comment:
Can you add this dependency to `packages/dendrites/setup.cfg` ?

Reply:
can I list `scipy.stats` as the dependency, or does it have to be `scipy`?

Reply:
just `spicy`

Reply:
done.